docs/pbc/all/payment-service-provider/202307.0/base-shop/third-party-integrations/computop/install-and-configure-computop.md

```diff
@@ -60,7 +60,7 @@ $config[KernelConstants::DOMAIN_WHITELIST] = [
 	...
     'www.computop-paygate.com', // A trusted Computop domain, required for redirects to third-party services.
 ];
-$config[SessionConstants::YVES_SESSION_COOKIE_SAMESITE] = 'none'; // Allows to redirect customers from Computop back to the shop via a `POST` request.
+$config[SessionConstants::YVES_SESSION_COOKIE_SAMESITE] = Cookie::SAMESITE_NONE; // Allows to redirect customers from Computop back to the shop via a `POST` request. (https://developer.computop.com/display/EN/Third-party-cookies+-+Browser+cookies+and+session+handling)
```

Comment:
Why it is not `SAMESITE_LAX`? Does it mean that `computop` does not work now on suite-nonsplit?

Reply:
It should work if the customer follows the documentation. Here I Just changed the string to const and added a link to their documentation where they advice to use `none` only.